errors/B0003.md

```diff
@@ -11,98 +11,52 @@ fn main() {
     App::new()
         .add_plugins(DefaultPlugins)
         .add_systems(Startup, setup)
-        .add_systems(Update, despawning)
-        .add_systems(Update, use_entity.after(despawning))
+        .add_systems(Update, (use_0_and_despawn_1, use_1_and_despawn_0))
         .run();
 }
 
 #[derive(Resource)]
-struct MyEntity(Entity);
+struct MyEntities([Entity; 2]);
```

Comment:
I think this would be better as 2 tuple arguments instead of an array.
```suggestion
struct MyEntities(Entity, Entity);
```

Reply:
using a tuple means accessing the entities from a resource with `my_entities.0 .0` which is less pretty than the array...

Reply:
If you do:
```rs
struct MyEntities(Entity, Entity);
```
instead of:
```rs
struct MyEntities((Entity, Entity));
```
then you can do `my_entities.0` / `my_entities.1` instead of `my_entities.0.0` or `my_entities.0[0]`

Reply:
ha right 😄